### DISCUSSION TITLE
error password user token with  wsdl no work with robust password

### DISCUSSION THREAD

**Discussion by jpertuz1** [Keyfactor/ejbca-ce#493] [General]
Greetings. I have a problem with the generation of the pkcs12 token key using the wsdl webservices.

The generation works well. But it doesn't work when I pass more robust keys example: "Aing12test"

getting the following error:

 body: '<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"><soap:Body><soap:Fault><faultcode>soap:Server</faultcode><faultstring>Got request for user with invalid password: V28152139.</faultstring><detail><ns1:EjbcaException xmlns:ns1="http://ws.protocol.core.ejbca.org/"><errorCode xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ns2="http://ws.protocol.core.ejbca.org/" xsi:type="ns2:errorCode"><internalErrorCode>LOGIN_ERROR</internalErrorCode></errorCode></ns1:EjbcaException></detail></soap:Fault></soap:Body></soap:Envelope>'
}

The password policy in the Entity is free.
In short a key of type: 123456 or A123456 works. But it doesn't work with a more robust key.

work fine: 
xmlns:tns="http://ws.protocol.core.ejbca.org/"><soap:Body><tns:editUser><arg0><caName>AC-MY</caName><endEntityProfileName>Ent-SignDirect</endEntityProfileName><certificateProfileName>Signer</certificateProfileName><email>usertest@web.com</email><extendedInformation><name>name</name><value>San</value></extendedInformation><username>V6221522</username><sendNotification>false</sendNotification><password>V6221522</password><keyRecoverable>false</keyRecoverable><clearPwd>false</clearPwd><tokenType>P12</tokenType><subjectDN>CN=V6221522,O=Link,C=VE,EMAILADDRESS=usertest@web.com</subjectDN><status>10</status></arg0></tns:editUser></soap:Body></soap:Envelope>',

**Comment by primetomas** [comment]
It works for me. Is it something with your client?

./ejbcaClientToolBox.sh EjbcaWsRaCli edituser p12user Aing12test false "CN=p12user" NULL NULL ManagementCA 1 P12 NEW EMPTY ENDUSER NULL

./ejbcaClientToolBox.sh EjbcaWsRaCli pkcs12req p12user Aing12test 2048 RSA NONE .

openssl pkcs12 -in ./p12user.p12 
(using Aing12test as password)
